packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -251,6 +253,7 @@ const MultiSelect = (props: Props) => {
         shortcuts = false,
         style,
         className,
+        "aria-label": ariaLabel,
```

Comment:
(non-blocking) docs: One thing I realized with the aria attributes is that they don't automatically show up in the Storybook component docs 😢 : https://5e1bf4b385e3fb0020b7073c-oodutnedcb.chromatic.com/?path=/docs/packages-dropdown-multiselect--docs 

We could explicitly add it to the story argTypes so they show up in the table! This can also be part of other work, since this is common to all components that extend the `AriaProps` type

Reply:
That is a great idea! I can open an issue for it, unless there already is one!